app/controllers/games_controller.rb

```diff
@@ -27,6 +27,18 @@ def index
     else
       @games = @games.sort_location
     end
+    # Don't show games that current user is already rsvped to
+    user_games = UserGamePair.all
+    remove_games = []
+    @games.each do |g|
+      user_games.each do |ug|
+        if ug.game_id == g.id and ug.user_id == 1
```

Comment:
We might run into some issues if we check "ug.user_id == 1" (for example, if we somehow have the user_id when we populate = 2, and I just dont think its necessary.